tgui/packages/tgui/interfaces/TurbineComputer.js

```diff
@@ -27,8 +28,8 @@ export const TurbineComputer = (props, context) => {
               />
               <Button
                 icon="sync"
```

Comment:
You may also want to change this icon, you can view the potential icons here https://fontawesome.com/search?q=xmark&o=r. I recommend xmark
```suggestion
                icon="xmark"
```